src/components/UserBackgroundImage.js

```diff
@@ -42,11 +58,31 @@ const UserBackgroundImage = ({ user }) => {
     backgroundImage: { timestamp: backgroundImageTimestamp, imageURL },
     id: userId,
   } = user
+  const [backgroundImages, setBackgroundImages] = useState([
+    { imageURL, preloaded: true },
+  ])
+
+  const fireOnLoad = () => {
+    setBackgroundImages((prevState) => [
+      prevState[prevState.length - 2],
```

Comment:
Is there a case where `prevState` might be length 1? For example, if we disable SSR for the page and the img `onload` fires for the original image without any second image? If so, let's add a test and handle that case so we're not pushing `undefined` into this state.

Reply:
ooo yeah if we disable ssr and page isn't in the cache.  good catch!